src/types/resolveConstantValue.ts

```diff
@@ -29,6 +29,8 @@ function reduceIntImpl(ast: ASTExpression): bigint {
             return l & r;
         } else if (ast.op === "|") {
             return l | r;
+        } else if (ast.op === "^") {
+            return l ^ r;
```

Comment:
Let's create a `feature-const-eval` test file and add some tests for constant evaluation there. This most likely will work but we need to make sure we cover the case for negative numbers at least

Reply:
Actually, we already have src/test/features/constants.tact, we just need to add some tests there